components/mdns/mdns.c

```diff
@@ -2383,6 +2383,53 @@ static void _mdns_send_bye(mdns_srv_item_t **services, size_t len, bool include_
     }
 }
 
+/**
+ * @brief  Send bye for particular subtype
```

Comment:
or list of subtypes...
```suggestion
 * @brief  Send bye for particular subtypes
```